### PR TITLE
MR: Implement pypowsybl to file logger [issues 45 and 55] to main

### PR DESCRIPTION
**Pull request by mr0321**
#45 if load_data_from_local_storage is set to true, then checks if folder with files exists in ENTSOE_EXAMPLES_LOCAL, if not then downloads and extracts zip file from ENTSOE_EXAMPLES_EXTERNAL. If folder is found then using the file name the file is packaged to similar shape as it would have come from OPDM
#55 For getting pyposybl logs, a separate instance of PyPowsyblLogGatherer must be called and set up (what and where to store). 
Storing depends on the policy set (either all entries or entries if specific log level was caught). Note that this does not catch java errors as these appear in usual log and are reported separately.

### CODE REVIEW

File: emf/loadflow_tool/helper.py [hunk 1]
```diff
@@ -153,15 +154,24 @@ def load_model(opdm_objects: List[dict]):
...
     return model_data
 
 
 def opdmprofile_to_bytes(opdm_profile):
+    # Temporary fix: input data (['opdm:Profile']['DATA']) can be a zip file, figure it out and extract
+    # before proceeding further
     data = BytesIO(opdm_profile['opdm:Profile']['DATA'])
-    data.name = opdm_profile['opdm:Profile']['pmd:fileName']
+    file_name = opdm_profile['opdm:Profile']['pmd:fileName']
+    if zipfile.is_zipfile(data) and not file_name.endswith('.zip'):
```
Comment (Haigutus) [inline emf/loadflow_tool/helper.py:168]: This check and workaround should not be handled here

**Review by Haigutus — CHANGES_REQUESTED** [review]

File: emf/loadflow_tool/helper.py (hunk 1 above)
Comment (mr0321) [reply emf/loadflow_tool/helper.py:168]: Removed in the branch mr-66-merge-for-RMM

**Review by Haigutus — CHANGES_REQUESTED** [review]
Leave here only logger changes, exclude validator changes

**Comment by mr0321** [comment]
Refactoring to different branches and pull requests

**Pull request closed without merging**